Add SSH support for DDEV Playwright (#99)
inpsyde/reusable-workflows: docs/ddev-playwright.md

| `ENV_VARS`           | Additional environment variables as a JSON formatted object.                              |
| `DDEV_ENV_VARS`      | Additional environment variables for DDEV as a JSON formatted object.                     |
| `NGROK_AUTH_TOKEN`   | The auth token for Ngrok. If not provided, skips Ngrok.                                   |
| `SSH_KEY`            | The private SSH key allowing access remote destinations.                                  |

> | `SSH_KNOWN_HOSTS`    | To fill `known_hosts` file for SSH agent.                                                 |

Reviewer: ```suggestion
| `SSH_KNOWN_HOSTS`    | SSH hosts to be set in the `known_hosts` file.                                            |
```
